### PR TITLE
Feat: assignment6 oops implementation added

### PR DESCRIPTION
**Pull request by Dharanash15**

### CODE REVIEW

File: src/Assignment6/BankingSystem/Program.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            try
+            {
+                Console.Write("Enter your Account Number : ");
+                accountNumber = Convert.ToInt32(Console.ReadLine());
+                Console.Write("Select your account accountType: 1.Saving account 2.Checking account : ");
+                accountType = Console.ReadLine();
+
+                if (accountType == "1")
+                {
+                    SavingsAccount savingsAccount = new SavingsAccount(accountNumber);
```
Comment (arun-soliton) [inline src/Assignment6/BankingSystem/Program.cs:29]: `Program.cs` can have only logics like creating respective classes based on user options and calling the respective methods. Try making the change.

**Review by arun-soliton — CHANGES_REQUESTED** [review]
Kindly make changes based on the comments.

File: src/Assignment6/ShapeHierarchy/Circle.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        /// <summary>
+        /// Method to calculate area from the given input
+        /// </summary>
+        /// <returns>Returns the calculated area as double</returns>
+        public override double CalculateArea()
+        {
+            Console.Write("Enter the Radius of Circle in cm: ");
+            int radius = Convert.ToInt32(Console.ReadLine());
+            return Math.PI * radius * radius;
```
[the remark this answers is not in the document]
Comment (Dharanash15) [reply src/Assignment6/ShapeHierarchy/Circle.cs:39]: I create a new method to resolve it.

File: src/Assignment6/ShapeHierarchy/Circle.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        /// <summary>
+        /// Gets or sets shape by child classes
+        /// </summary>
+        /// <value>
+        /// string value
+        /// </value>
+        private string Shape { get; set; }
+
+        /// <summary>
+        /// Method to calculate area from the given input
```
[the remark this answers is not in the document]
Comment (Dharanash15) [reply src/Assignment6/ShapeHierarchy/Circle.cs:32]: resolved

File: src/Assignment6/ShapeHierarchy/Rectangle.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            Console.Write("Enter the Length of Rectangle in cm: ");
+            int length = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Enter the Breath of Rectangle in cm: ");
+            int breath = Convert.ToInt32(Console.ReadLine());
+            return length * breath;
+        }
+
+        /// <summary>
+        /// Method declared to print details of shape in child class
```
[the remark this answers is not in the document]
Comment (Dharanash15) [reply src/Assignment6/ShapeHierarchy/Rectangle.cs:39]: resolved

File: src/Assignment6/ShapeHierarchy/Rectangle.cs [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        /// </summary>
+        /// <param name="area">area of a given shape</param>
```
[the remark this answers is not in the document]
Comment (Dharanash15) [reply src/Assignment6/ShapeHierarchy/Rectangle.cs:41]: resolved

**Pull request merged**